x/programs/rust/examples/token/src/lib.rs

```diff
@@ -108,3 +108,183 @@ pub fn get_balance(program: Program, recipient: Address) -> i64 {
         .get(StateKey::Balance(recipient).to_vec())
         .unwrap_or_default()
 }
+
+#[cfg(test)]
+mod tests {
+    use std::env;
+
+    use serde::{Deserialize, Serialize};
+    use wasmlanche_sdk::simulator::{Operator, Require, ResultAssertion};
+    #[derive(Debug, Serialize, Deserialize)]
+    struct PlanResponse {
+        id: u32,
+        result: PlanResult,
+        error: Option<String>,
+    }
+
+    #[derive(Debug, Serialize, Deserialize)]
+    struct PlanResult {
```

Comment:
Should `PlanResponse/PlanResult` be defined in a standard place?

Reply:
yeah was testing out the api will move to sdk